src/Features/CSharpTest/ExtractMethod/ExtractLocalFunctionTests.cs

```diff
@@ -4872,9 +4872,9 @@ public static void Main(string[] args)
         }
 
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsExtractLocalFunction)]
-        public async Task TestMissingInGoto()
+        public async Task TestOnGoto()
         {
-            await TestMissingInRegularAndScriptAsync(
+            await TestInRegularAndScriptAsync(
```

Comment:
no real reason to block this.  like with many cases, we can let this go through and let user deal with fallout.